File: gcc/testsuite/gfortran.dg/gomp/loop-transforms/unroll-3.f90
! { dg-additional-options "-fdump-tree-omp_transform_loops" }
! { dg-additional-options "-fdump-tree-original" }

subroutine test1
  implicit none
  integer :: i
  !$omp unroll full
  do i = 1,10
     call dummy(i)
  end do
end subroutine test1

! Loop should be removed with 10 copies of the body remaining

! { dg-final { scan-tree-dump-times "dummy" 10 "omp_transform_loops" } }
! { dg-final { scan-tree-dump "#pragma omp loop_transform" "original" } }
! { dg-final { scan-tree-dump-not "#pragma omp" "omp_transform_loops" } }

File: gcc/testsuite/c-c++-common/gomp/loop-transforms/unroll-3.c
/* { dg-additional-options "-fdump-tree-omp_transform_loops" }
 * { dg-additional-options "-fdump-tree-original" } */

extern void dummy (int);

void
test1 ()
{
  int i;
#pragma omp unroll full
  for (int i = 0; i < 10; i++)
    dummy (i);
}

 /* Loop should be removed with 10 copies of the body remaining
  * { dg-final { scan-tree-dump-times "dummy" 10 "omp_transform_loops" } }
  * { dg-final { scan-tree-dump "#pragma omp loop_transform" "original" } }
  * { dg-final { scan-tree-dump-not "#pragma omp" "omp_transform_loops" } } */
